web/help/GuideUnit1.md

```diff
@@ -1896,12 +1894,37 @@ y axis, since those are the angles in between the numbers -90 and 90.  But the
 expression `arc(270, 90, 5)` covers the angles to the left of the y axis,
 because those are the numbers between 270 and 90.
 
+To see a half circle with radius 5 drawn on the corrdinate plane, use this code:
+
+~~~~~ . clickable
+program   = drawingOf(halfCircle & coordinatePlane)
+halfCircle=arc(-90,90,5)
```

Comment:
Minor nit: I'd like to encourage students to use spaces around the equal sign, so let's at least model that for them.

Reply:
Gotcha